testsuite/features/support/constants.rb

```diff
@@ -1684,4 +1682,27 @@
   fake-base-channel-suse-like
   fake-base-channel-i586
   test-base-channel-x86_64
+  sle-product-sles15-sp4-pool-x86_64
```

Comment:
review if these can be empty if not in gh validation... can we filter only for ghvalidation??

Reply:
review 

```
  elsif result.include?('repo size: 0')
    if EMPTY_CHANNELS.include?(channel)
      sync_status = true
```

what about a different constant??

Reply:
fixed by adding a package on each repo